pkg/list/format/table.go

```
@@ -0,0 +1,288 @@
+func extractValueKeys(data map[string]interface{}, stackKeys []string) []string {
+	var valueKeys []string
+
+	for _, stackName := range stackKeys {
+		stackData := data[stackName]
+
+		switch typedData := stackData.(type) {
+		case map[string]interface{}:
+			for k := range typedData {
+				valueKeys = append(valueKeys, k)
+			}
+			break
```

golangci-lint (1.64.8)

[error] S1023: redundant break statement

(gosimple)